main(demolink,report link)

Nyaroslav opened the pull request.

irenhh commented on src/style.css:
```diff
@@ -0,0 +1,16 @@
+  margin-bottom: 20px;
+}
+
+input,
+div {
+  margin-bottom: 10px;
+}
+
+fieldset input:last-child {
+  margin-bottom: 0;
+}
+
+input[type="color"] {
+  margin-top: 4px;
+}
```
use classes

irenhh commented on src/style.css:
```diff
@@ -2,15 +2,15 @@
-input[type="color"] {
+.pixel-perfect {
   margin-top: 4px;
 }
```
avoid this thing, if you need this px, you must have done something else wrong

Nyaroslav committed:
added layout

Nyaroslav committed:
fixed code structure

Nyaroslav committed:
demo link

Nyaroslav committed:
div to label and [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)

Nyaroslav committed:
- [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Nyaroslav.github.io/layout_html-form/report/html_report/)

etojeDenys commented on src/index.html:
```diff
@@ -1,17 +1,209 @@
+           
+      </fieldset>
+    </form>
+    <button 
+      class="button" 
+      type="submit"
+    >Submit</button>
```
```suggestion
    <button 
      class="button" 
      type="submit"
    >
      Submit
    </button>
```

etojeDenys reviewed (changes requested):
please, add links to your PR
